pkg/assembler/graphql/schema/pkgEqual.graphql

```diff
@@ -15,17 +15,17 @@
 
 # NOTE: This is experimental and might change in the future!
 
-# Defines a GraphQL schema for the CertifyPkg. It contains a list of packages that are similar
+# Defines a GraphQL schema for the PkgEqual. It contains a list of packages that are similar
 # along with the justification, origin and collector.
 """
-CertifyPkg is an attestation that represents when a package objects are similar
+PkgEqual is an attestation that represents when a package objects are similar
 
 packages (subject) - list of package objects
 justification (property) - string value representing why the packages are similar
 origin (property) - where this attestation was generated from (based on which document)
 collector (property) - the GUAC collector that collected the document that generated this attestation
 """
-type CertifyPkg {
+type PkgEqual {
```

Comment:
can we make changes to `examples/` as well?

Reply:
Will do!

Reply:
done